docs/tutorial.rst

```diff
@@ -178,6 +204,65 @@ If you run ``nox --list-sessions``, you'll see that this generates the following
 
 If you only want to run one of the parametrized sessions, see :ref:`running_paramed_sessions`.
 
+Giving friendly names to parametrized sessions
+----------------------------------------------
+
+The automatically generated names for parametrized sessions, such as ``tests(django='1.9', database='postgres')``, can be long and unwieldy to work with even with using :ref:`keyword filtering <opt-sessions-and-keywords>`. You can give parametrized sessions custom IDs to help in this scenario. These two examples are equivalent:
+
+::
```

Comment:
I'm partial to using `.. code-block:: python` in situations like this, but it's your call (I like to have [strict enforcement][1] on my code blocks though)

[1]: https://github.com/dhermes/bezier/blob/master/docs/custom_html_writer.py

Reply:
not a bad idea. In another pass sometime. I want to restructure some of these docs at some point anyway.